src/Microsoft.Data.Sqlite.Core/SqliteConnection.cs

```diff
@@ -220,7 +220,7 @@ public override void Close()
             }
 
             Transaction?.Dispose();
-            _db.Dispose();
```

Comment:
We could, but it would be a no-op since `raw.sqlite3_close_v2` marks the `sqlite3` object as already disposed.